src/reports/suite.js

```diff
@@ -0,0 +1,90 @@
+export default class SuiteReport {
+	constructor( name, parentSuite ) {
+		this.name = name;
+		this.fullName = parentSuite ? parentSuite.fullName.concat( name ) : [];
```

Comment:
Ack, just realized this is an array, not a string. Please disregard. (Replying to the right comment this time)